File: FrontEnd/src/app/features/sales/invoices/classes/view-models/list/invoice-list-vm.ts
import { InvoiceListAadeVM } from './invoice-list-aade-vm'
import { SimpleEntity } from '../../../../../../shared/classes/simple-entity'

export interface InvoiceListVM {

    customer: SimpleEntity
    date: SimpleEntity
    destination: SimpleEntity
    documentType: SimpleEntity
    batch: string
    grossAmount: number
    invoiceId: string
    invoiceNo: number
    formattedDate: string
    isEmailPending: boolean
    isEmailSent: boolean
    ship: SimpleEntity
    shipOwner: SimpleEntity
    aade: InvoiceListAadeVM

}

File: API/Features/Sales/Invoices/ViewModels/List/InvoiceListVM.cs
using API.Infrastructure.Classes;

namespace API.Features.Sales.Invoices {

    public class InvoiceListVM {

        public string InvoiceId { get; set; }
        public string Date { get; set; }
        public int InvoiceNo { get; set; }
        public SimpleEntity Customer { get; set; }
        public SimpleEntity Destination { get; set; }
        public SimpleEntity DocumentType { get; set; }
        public SimpleEntity Ship { get; set; }
        public SimpleEntity ShipOwner { get; set; }
        public decimal GrossAmount { get; set; }
        public bool IsEmailPending { get; set; }
        public bool IsEmailSent { get; set; }
        public InvoiceListAadeVM Aade { get; set; }

    }

}